### DISCUSSION TITLE
The meanings of parameters: smearing_method = fd

### DISCUSSION THREAD

**Discussion by scott-5** [deepmodeling/abacus-develop#1761] [General]
When smearing_method is fd, smearing_sigma is temperature as doc writes.

Supposing smearing_method = fd and smearing_sigma = 273, I wonder output energies are energies at that 273K ? And if I want to get Gibbs energy, just need add above energies and ZPE additionally?

Or it will extrapolate energies from 273K to 0K ?

**Comment by Satinelamp** [comment]
smearing is a hypothesis that the electrons distribution on the bands, which is not relevant to the real temperature of your system.

**Reply by scott-5** [reply]
Thanks.
